ccm_web/client/src/utils/handleErrors.ts

```diff
@@ -48,8 +48,8 @@ const handleErrors = async (resp: Response): Promise<void> => {
       console.error(text)
       throw new NotFoundError()
     default:
-      console.error(await resp.text())
-      throw new Error(resp.statusText)
+      text = (JSON.parse(await resp.text()) as Error).message
```

Comment:
Is there some scenario where this won't work?
resp.statusText was blank in my use case, so i changed it to parse the message from the json object in the text.

Reply:
Backend will always send response text I don't know why it was blank, what was the status code did you receive for a blank statusText

Reply:
I believe it was 400.

Reply:
This `resp.text()` is not holding any info about the cause of the error that BE sends  but instead this is more useful `resp.json()`

@ssciolla `resp.text() and resp.statusText()` is the default message from JS library have based on the status code. Why did you prefer this string over the BE sent message.

Reply:
@ssciolla never mind I got confused.